### ISSUE TITLE
Keyboard shortcut does not open Find and Replace dialog in read-only mode

### ISSUE THREAD

**Issue by Acrophost** [ckeditor/ckeditor5#16297]
## 📝 Provide detailed reproduction steps (if any)

1. Open an editor including Find and Replace plugin in read-only mode
2. Use shortcut to open the Find and Replace dialog - CMD+F (CTRL+F)

### ✔️ Expected result

The find and replace dialog will open, but the replace part will be disabled.

### ❌ Actual result

Instead of opening find and replace dialog, the browser's find bar opens instead.
The Find and Replace button in toolbar is not disabled and when opened via mouse, the replace functionality is disabled (as it should be)

https://github.com/ckeditor/ckeditor5/assets/66953380/74ebdfcb-38d0-42b2-bea5-326626b03dda

## 📃 Other details

* Browser: all
* OS: all
* First affected CKEditor version: was present in 38.0.0, probably even longer

---

If you'd like to see this fixed sooner, add a 👍 reaction to this post.

**Comment by Mati365** [comment]
@Acrophost I verified this issue and it looks like current behaviour is the correct one. `ctrl + f` listener is not being executed because we do not focus editable in readonly mode (and focus stays at the `body` element). The problem is that we cannot listen to `ctrl + f` on  `body` because it'd open multiple find&replace popups in multiple editors at the same time after pressing such combination. 

If it's a problem for particular integrations(where editor is the only one element on the page) it can be fixed on integration side by listening `body` and toggling `enableReadOnlyMode`.

At this moment I'm closing this ticket.
